language/Spanish_Chile/strings.po

```diff
@@ -0,0 +1,6575 @@
+#, c-format
+msgid "%1$1.2f"
+msgstr "%1$1.2f"
+
+#: src/frontend/mame/ui/ui.cpp:2033
+#, c-format
+msgid "Crosshair Scale X %1$1.3f"
+msgstr "Escala X de punto de mira %1$1.3f"
+
+#: src/frontend/mame/ui/ui.cpp:2033
+#, c-format
+msgid "Crosshair Scale Y %1$1.3f"
+msgstr "Escala Y de punto de mira %1$1.3f"
+
+#: src/frontend/mame/ui/ui.cpp:2050
+#, c-format
+msgid "Crosshair Offset X %1$1.3f"
+msgstr "Desplazamiento X de punto de mira %1$1.3f"
+
+#: src/frontend/mame/ui/ui.cpp:2050
+#, c-format
+msgid "Crosshair Offset Y %1$1.3f"
+msgstr "Desplazamiento Y de punto de mira %1$1.3f"
+
+#: src/frontend/mame/ui/ui.cpp:2137
+msgid "**Error saving ui.ini**"
+msgstr "**Error guardando «ui.ini»**"
+
+#: src/frontend/mame/ui/ui.cpp:2196
+#, c-format
+msgid "**Error saving %s.ini**"
+msgstr "Error guardando «%s.ini»**"
+
+#: src/frontend/mame/ui/ui.cpp:2200 src/frontend/mame/ui/miscmenu.cpp:773
+msgid ""
+"\n"
+"    Settings saved    \n"
+"\n"
+msgstr ""
+"\n"
+"    Ajustes guardados    \n"
+"\n"
+
+#: src/frontend/mame/ui/selsoft.cpp:510 src/frontend/mame/ui/selgame.cpp:318
+#, c-format
+msgid ""
+"%s\n"
+" added to favorites list."
+msgstr ""
+"%s\n"
+" agregado a la lista de favoritos."
+
+#: src/frontend/mame/ui/selsoft.cpp:515 src/frontend/mame/ui/selgame.cpp:323
+#: src/frontend/mame/ui/selgame.cpp:329
+#, c-format
+msgid ""
+"%s\n"
+" removed from favorites list."
+msgstr ""
+"%s\n"
+" removido de la lista de favoritos."
+
+#: src/frontend/mame/ui/selsoft.cpp:576
+msgid "[Start empty]"
+msgstr "[Iniciar vacío]"
+
+#: src/frontend/mame/ui/selsoft.cpp:576
+msgid "[Use file manager]"
+msgstr "[Usa administrador de archivos]"
+
+#: src/frontend/mame/ui/selsoft.cpp:756
+#, c-format
+msgid "%1$s %2$s ( %3$d / %4$d software packages )"
+msgstr "%1$s %2$s ( %3$d / %4$d paquetes de software )"
+
+#: src/frontend/mame/ui/selsoft.cpp:757
+#, c-format
+msgid "%1$s - select software"
+msgstr "%1$s - seleccionar software"
+
+#: src/frontend/mame/ui/selsoft.cpp:762 src/frontend/mame/ui/selgame.cpp:1092
+#, c-format
+msgid "%1$s: %2$s - Search: %3$s_"
+msgstr "%1$s: %2$s - Buscar: %3$s_"
+
+#: src/frontend/mame/ui/selsoft.cpp:764 src/frontend/mame/ui/selgame.cpp:1094
+#, c-format
+msgid "Search: %1$s_"
+msgstr "Buscar: %1$s_"
+
+#: src/frontend/mame/ui/selsoft.cpp:771
+#, c-format
+msgid "Software list/item: %1$s:%2$s"
+msgstr "Lista/elemento de software: %1$s:%2$s"
+
+#: src/frontend/mame/ui/optsmenu.cpp:45 src/frontend/mame/ui/selgame.cpp:462
+msgid "General Settings"
+msgstr "Ajustes generales"
+
+#: src/frontend/mame/ui/optsmenu.cpp:77 src/frontend/mame/ui/sndmenu.cpp:30
+msgid "Sound Options"
+msgstr "Opciones de sonido"
+
+#: src/frontend/mame/ui/optsmenu.cpp:80
+msgid "Input Assignments"
+msgstr "Asignaciones de entrada"
+
+#: src/frontend/mame/ui/optsmenu.cpp:83 src/frontend/mame/ui/miscmenu.cpp:888
+msgid "Plugins"
+msgstr "Complementos"
+
+#: src/frontend/mame/ui/optsmenu.cpp:85
+msgid "Input Devices"
+msgstr "Dispositivos de entrada"
```

Comment:
This is referring to input devices, controllers, keyboards, mice, etc. The term used here is correct, but there might be a better option.